x11-misc/bumblebee/bumblebee-9999.ebuild

```diff
@@ -17,6 +18,8 @@ else
 	S="${WORKDIR}/Bumblebee-${COMMIT}"
 fi
 
+inherit ${SCM} autotools multilib readme.gentoo-r1 systemd user
+
 DESCRIPTION="Service providing elegant and stable means of managing Optimus graphics chipsets"
 HOMEPAGE="http://bumblebee-project.org https://github.com/Bumblebee-Project/Bumblebee"
 
```

Comment:
move `DOC_CONTENTS` just to before the call of `readme.gentoo_create_doc`, and make it `local`